tests/mobly/logger_test.py

```diff
@@ -23,9 +24,9 @@ class LoggerTest(unittest.TestCase):
     """Verifies code in mobly.logger module.
     """
     def test_epoch_to_log_line_timestamp(self):
-        # actual_stamp = logger.epoch_to_log_line_timestamp(1469134262116)
-        # self.assertEqual("07-21 13:51:02.116", actual_stamp)
-        pass
+        actual_stamp = logger.epoch_to_log_line_timestamp(1469134262116, 
```

Comment:
It looks like there's a trailing space at the end of this line.
Please remove.
You can use linter _yapf_ to take care of formatting.
`yapf -i --style pep8 <file path>